docs/configuration-reference.md

```diff
@@ -3742,3 +3744,78 @@ Full reference of options:
     # Maps to collector.logical_disk.volume-blacklist in windows_exporter
     [blacklist: <string> | default=".+"]
 ```
+
+### kafka_exporter_config
+
+The `kafka_exporter_config` block configures the `kafka_exporter`
+integration, which is an embedded version of [`kafka_exporter`](https://github.com/davidmparrott/kafka_exporter).
+This allows for the collection of Kafka Lag metrics and exposing them as Prometheus metrics.
+
+Full reference of options:
+
+```yaml
+
```

Comment:
Are any of these fields required? If so, remove the surrounding `[]` from the field. I'd imagine kafka_uris is _probably_ required, right?